client/post-editor/media-modal/detail/detail-preview-videopress.jsx

```diff
@@ -90,6 +128,34 @@ class EditorMediaModalDetailPreviewVideoPress extends Component {
 		}
 	}
 
+	play() {
+		if ( ! this.player || ! this.player.state ) {
+			return;
+		}
+
+		if ( typeof this.player.state.play === 'function' ) {
+			this.player.state.play();
+		}
+	}
+
+	pause() {
+		if ( ! this.player || ! this.player.state ) {
+			return;
+		}
+
+		if ( typeof this.player.state.pause === 'function' ) {
+			this.player.state.pause();
+		}
```

Comment:
these are inherently unsafe depending on nested property access and they can kill the execution thread

check out [lodash.invoke()](https://kapeli.com/dash_share?docset_file=Lo-Dash&docset_name=Lodash&path=lodash.com/docs/4.17.html%23invoke&platform=lodash&repo=Main&source=lodash.com/docs/4.17.4&version=4.17.4) for a safe executor…

```js
invoke( this, 'player.state.pause' )
```